doc/getting_started.md

```diff
@@ -0,0 +1,83 @@
+# Getting started
+
+This document will guide through setting up PyDotBot connected to a DotBot
```

Comment:
This document will help guide you through the setup of PyDotBot connected to a DotBot